operations/iuf/IUF.md

```diff
@@ -204,12 +209,13 @@ options:
                         options also specified. The file is named via the `-i` argument. The command exits once the
                         file has been created.
   -a ACTIVITY, --activity ACTIVITY
-                        Activity name. Must be a unique identifier. Activity names must only contain lowercase letters (a-z),
+                        Activity name. Must be a unique identifier. Activity names must contain only lowercase letters (a-z),
                         numbers (0-9), periods (.), and dashes (-). Can also be set via the IUF_ACTIVITY environment
                         variable.
   -c CONCURRENCY, --concurrency CONCURRENCY
-                        Run Argo processes concurrently. Defaults to a fully concurrent model. To use at most N
-                        threads, specify `concurrency N`.
+                        During stage processing Argo runs workflow steps in parallel. By default up to 10 steps will be
```

Comment:
...up to 10 steps are executed...

Reply:
This text comes from the `iuf` command's `--help` output, so I don't want to change the doc without a code change first.